src/itemsList/ItemsList.tsx

```diff
@@ -38,6 +40,22 @@ export const ItemsList = (props: ItemsListProps) => {
   } = props
   const a11yAttrs = pickA11yProps<ItemsListProps>(props)
 
+  function isSingleChild(item: ChildrenType): item is ItemsListChild {
```

Comment:
You might want to try something like:

```
const items = Array.isArray(children) ? children : [children]
const filteredChildren = children.filter(Boolean)
```

That way you won't have to keep the next 10 lines

Reply:
Done, I had try this but was misusing isArray. Works fine!